cmd/start.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2017 the lostromos Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/wpengine/lostromos/crwatcher"
+	"github.com/wpengine/lostromos/printctlr"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	crdName      string
+	crdGroup     string
+	crdVersion   string
+	crdNamespace string
+	kubeFile     string
+	kubeMaster   string
+	inKube       bool
+)
+
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: `Start the server.`,
+	Run: func(command *cobra.Command, args []string) {
+		startServer()
```

Comment:
This feels like it should `command.Flags()` rather than globals for config (although looking at the code, that may not be idiomatic, feels like we're using globals when we don't have to)

Reply:
I don't entirely disagree with you. I based this pattern on how the guides for cobra and viper show you should handle flags using their tools. There are ways to store the results not in a var though, but looking at it, it felt more complex. I'm open to changing though.

Reply:
Fixed. Forgot to integrate with viper. 😞